airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -450,7 +450,6 @@ def get_md5hash(self, bucket_name, object_name):
         self.log.info('The md5Hash of %s is %s', object_name, blob_md5hash)
         return blob_md5hash
 
-    @CloudBaseHook.catch_http_exception
```

Comment:
Why this? I believe catch_http_exception was catching the exception and throwing AirflowException instead

Reply:
That's true, but original exception was hard to catch like:
```
try:
   create_smth()
except AlreadyExists:
   get_smth()
```